client/a8c-for-agencies/sections/purchases/licenses/license-actions/use-license-actions.tsx

```diff
@@ -0,0 +1,269 @@
+				isExternalLink: true,
+				isEnabled: type === 'wpcom',
+			},
+
+			{
+				name: translate( 'Edit site in WP Admin' ),
+				href: `${ license.siteUrl }/wp-admin/admin.php?page=jetpack#/dashboard`,
+				onClick: () => handleClickMenuItem( 'calypso_a4a_licenses_edit_site_click' ),
+				isExternalLink: true,
+				isEnabled: type === 'wpcom',
+			},
+
+			{
+				name: translate( 'Download' ),
+				onClick: () => {
+					handleClickMenuItem( 'calypso_a4a_licenses_download_click' );
+					downloadLicense( null, {
+						onSuccess: ( data ) => {
+							window.location.replace( data.download_url );
+						},
+						onError: ( error ) => {
+							dispatch( errorNotice( error.message ) );
+						},
+					} );
+				},
+				isExternalLink: false,
+				isEnabled:
+					type === 'regular' &&
+					license.hasDownloads &&
+					licenseState === LicenseState.Attached &&
+					licenseType === LicenseType.Partner,
+			},
+
+			{
+				name: translate( 'View site' ),
+				href: license.siteUrl ?? '',
+				onClick: () => handleClickMenuItem( 'calypso_a4a_licenses_view_site_click' ),
+				isExternalLink: true,
+				isEnabled:
+					type === 'regular' && licenseState === LicenseState.Attached && ! isPressableLicense,
+			},
+
+			{
+				name: translate( 'Debug site' ),
+				href: `https://jptools.wordpress.com/debug/?url=${ license.siteUrl }`,
+				onClick: () => handleClickMenuItem( 'calypso_a4a_licenses_debug_site_click' ),
+				isExternalLink: true,
+				isEnabled:
+					[ 'wpcom', 'regular' ].includes( type ) &&
+					licenseState === LicenseState.Attached &&
+					! isPressableLicense,
+			},
+
+			{
+				name: translate( 'Manage in Pressable' ),
+				href: 'https://my.pressable.com/agency/auth',
+				onClick: () => handleClickMenuItem( 'calypso_a4a_licenses_manage_in_pressable_click' ),
+				isExternalLink: true,
+				isEnabled:
+					type === 'regular' && licenseState === LicenseState.Attached && isPressableLicense,
+			},
+
+			{
+				name: translate( 'Upgrade' ),
+				href: isPressableLicense
+					? A4A_MARKETPLACE_HOSTING_PRESSABLE_LINK
+					: A4A_MARKETPLACE_HOSTING_WPCOM_LINK,
+				onClick: () => handleClickMenuItem( 'calypso_a4a_licenses_upgrade_click' ),
+				isExternalLink: false,
+				isEnabled:
+					[ 'wpcom', 'regular' ].includes( type ) &&
+					! isClientLicense &&
+					! isDevSite &&
+					( isPressableLicense || isWPCOMHostingLicense ),
+			},
+
+			{
+				name: translate( 'Revoke' ),
+				href:
+					type === 'wpcom'
+						? `https://wordpress.com/purchases/subscriptions/${ siteSlug }`
+						: undefined,
+				isExternalLink: type === 'wpcom',
+				onClick: () => handleClickMenuItem( 'calypso_a4a_licenses_revoke_click' ),
+				type: 'revoke',
+				isEnabled:
+					[ 'wpcom', 'regular' ].includes( type ) &&
+					canRevoke &&
+					( isChildLicense ? licenseState === LicenseState.Attached : true ) &&
+					licenseType === LicenseType.Partner,
+				className: 'is-destructive',
+				dialog:
+					type === 'regular'
```

Comment:
This should be for both [ 'wpcom', 'regular' ]

We could remove the check actually.